client/src/components/App.jsx

```diff
@@ -6,15 +6,96 @@ class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      booking: {}
+      listing: {},
+      guestTotal: 1,
+      guestAdults: 1,
+      guestChildren: 0,
+      guestInfants: 0
+    }
+  }
+
+  componentDidMount() {
+    fetch('/rooms/bookings/listings')
+    .then(res => res.json())
+    .then((result) => {
+      console.log(result[0]);
+      this.setState({
+        listing: result[0]
+      })
+    },
+    (err) => {
+      console.log('ERROR ON MOUNT: ' + err);
+    })
+  }
+
+  updateGuestTotal(type, guest) {
```

Comment:
The logic for this function is great, I have seen how your component works, and I gotta say. Good job Janelle!